src/modules/dashboard/components/CoinMachine/CoinMachine.css

```diff
@@ -1,5 +1,6 @@
 .main {
   display: block;
+  margin-bottom: 100px;
```

Comment:
Why 100px? I think it's very wrong...

Reply:
I've moved to using width and height with calc, otherwise main class is smaller than it's child grid, and margin's don't get applied on lower resolutions?

Hope it's good now, let me know. Thank you